src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -291,8 +292,12 @@ export class WfsSearch extends React.Component {
     } = this.props;
 
     const request = this.getCombinedRequests();
+    const fetchTime = new Date();
 
-    this.setState({fetching: true});
+    this.setState({
+      fetching: true,
+      latestRequestTime: fetchTime.getTime()
+    });
```

Comment:
Not sure, if it may be relevant here, but updating the state *might* be asynchronous so the check in `onFetchSuccess()` *might* fail in succession. It might be worthwhile to check to call the fetch in the callback from `setState()`.

Reply:
You're right, of course. Fixed this and the failing test in 0ff62fc.